packages/browser-crawler/package.json

```diff
@@ -59,6 +59,7 @@
         "@crawlee/types": "3.10.4",
         "@crawlee/utils": "3.10.4",
         "ow": "^0.28.1",
+        "playwright": "1.44.1",
```

Comment:
this one is wrong, playwright is optional peer dependency on this level, also we shouldn't constraint its version anyhow

also it feels weird that you would need to handle only playwright and not puppeteer, sounds like you were using that instead - both of them should be handled the same as in here

https://github.com/apify/crawlee/blob/master/packages/crawlee/package.json#L72-L83

Reply:
Agreed, backed out the erroneous addition and force pushed.

Reply:
can you add the peer dependencies? also the lockfile is outdated, you still have the playwright dependency there